config/prod.webpack.config.ts

```diff
@@ -12,7 +12,8 @@ plugins.push(
     federatedModules(
         {
             root: resolve(__dirname, '../'),
-            useFileHash: false
+            useFileHash: true,
```

Comment:
Should this be `false` on prod ?

Reply:
Ah, nevermind. True on prod, false on dev. :yawning_face:

Reply:
No, we want it true on prod to prevent stale assets to be served. We had a lot of issues with this in the past. It also can reveal invalid builds when beta assets are served on stable env and vice versa.